development/tools/pkg/release/release.go

```diff
@@ -35,26 +36,27 @@ func (c *creatorImpl) CreateNewRelease(ctx context.Context, relOpts *Options, ar
 	}
 
 	for _, artifact := range artifactNames {
-		if err = c.createReleaseArtifact(ctx, *release.ID, artifact, relOpts.Version); err != nil {
+		if err = c.createReleaseArtifact(ctx, *release.ID, artifact); err != nil {
 			return errors.Wrapf(err, "while creating release artifact: %s", artifact)
 		}
 	}
 
 	return nil
 }
 
-func (c *creatorImpl) createReleaseArtifact(ctx context.Context, releaseID int64, artifactName, folderName string) error {
+func (c *creatorImpl) createReleaseArtifact(ctx context.Context, releaseID int64, artifactName string) error {
 
-	fullArtifactName := folderName + "/" + artifactName
-
-	artifactData, size, err := c.storage.ReadBucketObject(ctx, fullArtifactName)
+	components, err := os.Open("installation/resources/components.yaml")
```

Comment:
Do not hardcode this directory. Let it be provided by a flag from entrypoint development/tools/cmd/githubrelease/main.go